tools/docker/build.sh

```diff
@@ -6,14 +6,11 @@
 # See LICENSE file for more details.
 ###############################################################
 
-run() {
-    echo "$*"
-    "$@" || exit $?
-}
-
 scriptdir="$(cd "${0%/*}"; pwd)"
 topdir="${scriptdir%/*/*/*}"
 
+. ${topdir}/prplMesh/tools/docker/functions.sh
```

Comment:
`${scriptdir}/functions.sh` would be more appropriate I think. (nitpick)

Same for all other scripts obviously.

Reply:
not all scripts are in the same path so it will become `${scriptdir}/../functions.sh` for some which I find less appealing. I think we can keep it as it is now, since `test_flows.sh` and `test_gw_repeater.sh` can be unified to a single test script which can move to the same directory as all others.